x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/relevance_tuning_form/relevance_tuning_item_content/relevance_tuning_item_content.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { EuiPageContent, EuiPageContentBody } from '@elastic/eui';
+
+import { SchemaTypes } from '../../../../../shared/types';
+
+import { Boost, SearchField } from '../../types';
+
+import { Boosts } from './boosts';
+import { TextSearchToggle } from './text_search_toggle';
+import { WeightSlider } from './weight_slider';
+
+import './relevance_tuning_item_content.scss';
+
+interface Props {
+  name: string;
+  type: SchemaTypes;
+  boosts?: Boost[];
+  field?: SearchField;
+}
+
+export const RelevanceTuningItemContent: React.FC<Props> = ({ name, type, boosts, field }) => {
+  return (
+    <>
+      <EuiPageContent hasShadow={false} className="relevanceTuningForm__itemContent">
```

Comment:
i think in this case `EuiPanel` might be better than `EuiPageContent`, since I don't think there's meant to be multiple `EuiPageContent`s per page

Reply:
Oh I thought it was alright? https://github.com/elastic/kibana/pull/83504#discussion_r526136961

Reply:
Ha! Past-Constance is super untrustworthy, don't believe her lies. Let me check with the EUI folks to see if they have some guidance here

Reply:
OK, official word from the EUI crew: one per page, which means I should go back in and change EngineOverview. Thanks for keeping me honest Jason!